trick_source/sim_services/VariableServer/VariableServerThread_commands.cpp

```diff
@@ -93,6 +99,36 @@ int Trick::VariableServerThread::var_add(std::string var_name, std::string units
     return(0) ;
 }
 
+// TODO: This should go somewhere else probably?
```

Comment:
It's used only here, personally I'd leave it. Maybe add a comment saying ```//Helper function for var_send_once``` so we can see that it's just for this file at a glance

Reply:
Added